main_server/local_server/server_admin/server_admin_javascript.js

```diff
@@ -4268,7 +4268,7 @@ function BMLT_Server_Admin()
         in_offset = parseInt(in_offset, 10); // Just to make sure.
         var format_line_tr = null;
 
-        var insertion_point = (g_formats_array.length + in_offset) * (g_langs.length + 1);
+        var insertion_point = (g_formats_array.length + in_offset) * (g_langs.length + 2);
```

Comment:
Will you explain the change on this line? I'm sure it's valid, I just want to understand.